src/backend/core/api/viewsets/entitlements.py

```
@@ -0,0 +1,95 @@
+            "organization": organization,
+            "service": service,
+            "service_subscription": service_subscription,
+        }
+
+        # This entitlement should always be resolved.
+        entitlements_data = {**AccessEntitlementResolver().resolve(entitlement_context)}
+
+        if service_subscription:
+            operator_data = OperatorSerializer(service_subscription.operator).data
+            entitlements = models.Entitlement.objects.filter(service_subscription=service_subscription)
+            print(entitlements)
```

Ruff (0.14.2)

`print` found

Remove `print`

(T201)